Fingerprints/ExtractContaminationFromArrays.wdl

```diff
@@ -0,0 +1,159 @@
+version 1.0
+
+workflow IdentifyContaminantArrayTest {
```

Comment:
I believe this is a wdl for testing an array based contamination branch we were working on a while ago.  I wonder if it belongs in this repo, since it's a pretty niche use case.  Maybe there is a way to generalize it a little more if we think we're likely to want to do similar types of tests in the future?

@meganshand I'd be interested to hear your thoughts on this type of workflow being in this repo.

Reply:
I'd imagine that we'll want to extract contaminants in the future, but I don't know about doing it for arrays. I'd vote for leaving it out of this PR for now until it's needed again, since that sounds easiest.